scripts/archive_migration_functions.sh

```diff
@@ -366,6 +383,12 @@ save_kbn() {
   #  node scripts/kbn_archiver.js --config x-pack/test/spaces_api_integration/security_and_spaces/config_basic.ts save x-pack/test/functional/fixtures/kbn_archiver/saved_objects/default_space --type search,index-pattern,visualization,dashboard,lens,map,graph-workspace,query,tag,url,canvas-workpad
 }
 
+load_my_kbn() {
```

Comment:
Should we keep these changes? is `my-es-archive.json` a file we are including in the kibana repo?

Reply:
We can drop this!  Good catch!